### DISCUSSION TITLE
Programm Partizipationsmeeting Dezember 2024 | Programme de la réunion participative decembre 2024

### DISCUSSION THREAD

**Discussion by christianheimann** [e-id-admin/governance-sounding-board#54] [Announcements]
Datum: 6. Dezember 2024
Zeit | Was
:-- | :--
ab 08.30 | Eintreffen zu Kaffee & Gipfeli
09.00 | Eröffnung
09:15 | Präsentation Stadt Zug
09:30 | Was braucht es für ein gutes Wallet? Panel mit Vertreterinnen und Vertretern von Deutschland, Italien, Procivis, Ubique und Bund
10:20 | Pause
10:50 | World Cafés zu Italien, Procivis, Ubique, AGr E-ID DVS/ANS, eLFA (Bund) Public Beta AMA (Bund)

**Mittagessen**

Zeit | Was
:-- | :--
14:00 | Neuigkeiten vom Bund
15:00 | Pause
15:30 | Präsentation "DVS4U: Integration des E-ID Ökosystems in kantonale und kommunale Systeme"
16:00 | Präsentation DIDAS
16:15 | Präsentation digitalswitzerland
16.30 | Closing & Apéro

**Wo?**
Physisch: Bundesamt für Informatik und Telekommunikation BIT, Eichenweg 3, 3052 Zollikofen

Online: Die Hauptpräsentationen werden online übertragen, Link folgt.

___________

Date: 6 décembre 2024
Heure | Sujet
:-- | :--
dès 08.30 | Arrivée pour café & croissants
09.00 | Ouverture
09:15 | Présentation de la ville de Zoug
09:30 | Que faut-il pour un bon wallet ? Panel avec des représentants d'Allemagne, d'Italie, de Procivis, d'Ubique et de la Confédération
10:20 | Pause
10:50 | World Cafés sur l'Italie, Procivis, Ubique, GrT e-ID ANS/DVS, eLFA (Confédération) Public Beta AMA (Confédération)

**Repas de midi**

 Heure | Sujet
:-- | :--
14:00 | Nouvelles de la Confédération
15:00 | Pause
15:30 | Présentation "DVS4U : Intégration de l'écosystème e-ID dans les systèmes cantonaux et municipaux"
16:00 | Présentation DIDAS
16:15 | Présentation digitalswitzerland
16.30 | Closing & Apéro

**Où?**
Physique: Office fédéral de l'informatique et de la télécommunication OFIT, Eichenweg 3, 3052 Zollikofen

En ligne: Les présentations principales seront diffusées en ligne, Lien à venir.

___________

Date: 6 December 2024
Time | What
:-- | :--
from 08.30 | Arriving for coffee and croissants
09.00 | Opening
09:15 | Presentation of the city of Zug
09:30 | What does a good wallet need? Panel with representatives from Germany, Italy, Procivis, Ubique and the federal government
10:20 | Break
10:50 | World Cafés on Italy, Procivis, Ubique, WGr IAM DVS/ANS, eLFA (federal government) Public Beta AMA (federal government)

**Lunch**

Time | What
:-- | :--
14:00 | News from the federal government
15:00 | Break
15:30 | Presentation "DVS4U: Integration of the e-ID ecosystem into cantonal and municipal systems"
16:00 | Presentation DIDAS
16:15 | Presentation digitalswitzerland
16.30 | Closing & drinks

**Where?**
In person: Federal Office of Information Technology, Systems and Telecommunication FOITT, Eichenweg 3, 3052 Zollikofen

Online: The main presentations will be streamed online, link to follow.

**Comment by damienbod** [comment]
Hi @christianheimann Hope you are well. Is the date fixed, if so, what is it?

Greetings Damien

**Reply by rolfrauschenbach** [reply]
December 6, 2024

**Comment by JakobSteingruber5** [comment]
Hi all
When is a detailed agenda available?
Greetings Jakob

**Reply by JakobSteingruber5** [reply]
Thank you
